src/plone/restapi/services/userschema/configure.zcml

```diff
@@ -0,0 +1,14 @@
+<configure
+    xmlns="http://namespaces.zope.org/zope"
+    xmlns:plone="http://namespaces.plone.org/plone"
+    xmlns:zcml="http://namespaces.zope.org/zcml">
+
+    <plone:service
+      method="GET"
+      name="@userschema"
+      for="Products.CMFCore.interfaces.ISiteRoot"
+      factory=".user.UserSchemaGet"
+      permission="zope2.View"
```

Comment:
Is this supposed to be public?

Reply:
At least we need it when registration for the site is opened (https://demo.plone.org/register), in other cases we could check for the user to be logged-in.